src/wallet/asyncrpcoperation_shieldcoinbase.h

```diff
@@ -23,29 +23,45 @@
 
 using namespace libzcash;
 
-struct ShieldCoinbaseUTXO {
-    uint256 txid;
-    int vout;
-    CScript scriptPubKey;
-    CAmount amount;
-};
+/**
+When estimating the number of coinbase utxos we can shield in a single transaction:
+1. An Orchard receiver is 9165 bytes.
```

Comment:
Why 9165? That's the size of a fully shielded Orchard transaction with 2 actions. The marginal size per receiver is much less.